### ISSUE TITLE
Write spec for club protocol

### ISSUE THREAD

**Issue by neatonk** [ReleaseClub/club-protocol#1]
See https://github.com/soundxyz/sound-protocol/blob/master/spec.md as an example.

**Comment by neatonk** [comment]
First cut is here: https://github.com/ReleaseClub/club-protocol/blob/9835de1cb4273a214ff080c4f1d041040f32a41d/spec.md

**Comment by josealonso** [comment]
@neatonk, what exactly are you trying to do ? Do you want to implement the contacts from the scratch ?

**Comment by neatonk** [comment]
Yes, exactly. The design has evolved a bit as you'll see when I add more to the spec. That said, there is a good deal that has not changed and we'll be able to pull in code from the existing contracts for that.

**Comment by neatonk** [comment]
The main differences are that (1) the club contract will refer to one or more curation contracts rather than being used for curation directly and (2) auth will be modular.

**Comment by neatonk** [comment]
@josealonso I will certainly be interested in hearing your questions and feedback once a bit more of the spec is on the page.

**Comment by josealonso** [comment]
Going to bed now. I'll tell you about a similar project I saw for musicians. Check this out: https://www.muse-dao.co/
